cmake/OctArgsPackages.cmake

```diff
@@ -43,12 +43,16 @@ set(CPACK_ARCHIVE_DEV_FILE_NAME         "${CPACK_PACKAGE_NAME}-dev_${CPACK_PACKA
 set(CPACK_ARCHIVE_EXAMPLES_FILE_NAME    "${CPACK_PACKAGE_NAME}-examples_${CPACK_PACKAGE_VERSION}-${PACKAGE_ARCHITECTURE}")
 
 set(CPACK_DEB_COMPONENT_INSTALL             "ON")
+set(CPACK_DEBIAN_PACKAGE_NAME               "lib${CPACK_PACKAGE_NAME}")
```

Comment:
I think if we want to have the lib prefix in package name also the file name should be modified.
Maybe the whole CPACK_PACKAGE_NAME should be switched to liboctargs? Or maybe the CPACK_DEBIAN_DEV_PACKAGE_NAME and other variables names shall be used and the CPACK_DEBIAN_DEV_FILE_NAME and related variables modified to achieve the required effect.

What is your opinion on that?

Reply:
I originally thought about that, but prefer to minimize the changes. More changes I introduce - higher the probability of the PR being rejected. Sad, but true.